<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pwmchannelui</class>
 <widget class="QFrame" name="pwmchannelui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="linename">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Channelname</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="slider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spin">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnRandom">
     <property name="toolTip">
      <string>Random value</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/icons/roll.png</normaloff>:/icons/roll.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnNoFade">
     <property name="toolTip">
      <string>Fading disabled</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/icons/dialog-cancel.png</normaloff>:/icons/dialog-cancel.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnFadeDown">
     <property name="toolTip">
      <string>Fade down</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnFadeUp">
     <property name="toolTip">
      <string>Fade up</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/icons/arrow-right.png</normaloff>:/icons/arrow-right.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
